src/content/en/fundamentals/instant-and-offline/web-storage/indexeddb-best-practices-app-state.md

```diff
@@ -0,0 +1,160 @@
+project_path: /web/_project.yaml
+book_path: /web/fundamentals/_book.yaml
+description: Learn best practices for syncing application state between IndexedDB an popular state management libraries
+
+{# wf_updated_on: 2017-06-08 #}
+{# wf_published_on: 2017-06-08 #}
+
+# IndexedDB Best Practices for Persisting Application State {: .page-title }
```

Comment:
I'd be open to changing it to "Best Practices for Persisting Application State with IndexedDB", but since it's not a how-to guide, I don't want to title it "Persisting Application State with IndexedDB". WDYT?